sources/academy/platform/getting_started/creating_actors.md

```diff
@@ -1,131 +1,183 @@
 
-## Build an actor {#build-an-actor}
+> Feel free to play around with the code and add some more features to it. For example, you can extract all the links from the page or extract all the images or completely change the logic of this template. Just keep in mind that this template uses [input schema](/academy/deploying-your-code/input-schema) defined in the `.actor/input_schema.json` file and linked to the `.actor/actor.json`. If you want to change the input schema, you need to change it in those files as well. Learn more about the Actor input and output [in the next page](/academy/getting-started/inputs-outputs).
 
-In order to run the actor, you've gotta build it. Luckily, that's only a button press away.
+## Build the Actor 🧱 {#build-an-actor}
+
+In order to run the Actor, you need to [build](/platform/actors/development/builds-and-runs/builds) it first. Click on the **Build** button at the bottom of the page or **Build now** button right under the code editor.
 
 ![Build the actor](./images/build-actor.png)
 
 After you've clicked the **Build** button, it'll take around 5-10 seconds to complete the build. You'll know it's finished when you see a green **Start** button.
 
 ![Start button](./images/start.png)
 
-And now we are ready to run the Actor. But before we do that, let's give the Actor some input by going to the `Input` tab.
+#### Fill the input {#fill-input}
+
+And now we are ready to run the Actor. But before we do that, let's give the Actor some input by going to the `Input` tab. 
+
+The input tab is where you can provide the Actor with some meaningful input. In this case, we'll be providing the Actor with a URL to scrape. For now, we'll use the prefilled value of [Apify website](https://apify.com/) (`https://apify.com/`).
+
+You can change the website you want extract the data from by simply changing the URL in the input field.
 
 ![Input tab](./images/actor-input-tab.png)
 
-## Run an actor {#run-an-actor}
+## Run the Actor {#run-the-actor}
 
-Once you have provided the Actor with some meaningful input, click **Start** and wait a few seconds... Do you see something that looks like this?:
+Once you have provided the Actor with some URL you want to extract the data from, click **Start** button and wait a few seconds. You should see the Actor run logs in the **Last run** tab.
 
 ![Actor run logs](./images/actor-run.png)
 
-If so, it worked! 🥳
+After the Actor finishes, you can preview or download the extracted data simply by clicking on the **Export X results** button.
+
+![Export results](./images/actor-run-dataset.png)
+
+And that's it! You've just created your first Actor and extracted data from a website 🎉.
+
+## Don't get stuck with tips 💡 {#dont-get-stuck-with-tips}
+
+If you ever get stuck, you can always click on the **Tips** button in the top right corner of the page. It will show you a list of tips that are relevant to the Actor development.
```

Comment:
```suggestion
If you ever get stuck, you can always click on the **Tips** button in the top right corner of the page. It will show you a list of tips that are relevant to the Actor's development.
```